src/mlang/driver.ml

```diff
@@ -65,8 +65,12 @@ let driver (files : string list) (debug : bool) (var_info_debug : string list) (
     if Mir_dependency_graph.check_for_cycle full_m_program.dep_graph full_m_program.program true
     then Errors.raise_error "Cycles between rules.";
     let mpp = Mpp_frontend.process mpp_file full_m_program in
-    let m_program = Mir_interface.reset_all_outputs full_m_program.program in
-    let full_m_program = Mir_interface.to_full_program m_program in
+    let full_m_program =
+      Mir_interface.to_full_program
+        (match function_spec with
+        | Some _ -> Mir_interface.reset_all_outputs full_m_program.program
+        | None -> full_m_program.program)
+    in
```

Comment:
So if I understand correctly, the new behavior is that if you don't pas `--function_spec=...` to the CLI arguments of Mlang, then it behaves as the old DGFiP compiler, assuming everything can be output and input? If that's the case, I would like this to be documented in `src/cli.ml` in the `function_spec` option documentation.